src/utils/base64.ts

```diff
@@ -0,0 +1,153 @@
+/* tslint:disable file-header */
```

Comment:
all that code was taken from https://gist.github.com/enepomnyaschih/54c437997f8202871278d0fdf68148ca (originally from: https://gist.github.com/enepomnyaschih/72c423f727d395eeaa09697058238727).

Note that the guy also made a lib out of it:  https://github.com/enepomnyaschih/byte-base64
And published it under the name "byte-base64"

I didn't want to take that dependency directly from the lib due to 3 things:
  1. This code looks simple enough. Adding it as a dependency would bring me more automatic updates but would be much more opaque (what code am I actually importing, which can be different than what is in its `src`). I prefer to stay in control here.
  2. There was a small issue where `{` characters would be wrongly interpreted as correct in his code. I notified him and fixed it here.
  3. It was not tolerant of un-padded strings (missing some `=` at the end). I added that.

To stay compliant with its license I kept it and did not include our copyright notice in that file.